### ISSUE TITLE
[feature]: random delay in typing

### ISSUE THREAD

**Issue by cltrudeau** [socsieng/sendkeys#59]
### What feature would you like to see in sendkeys?

Thanks for the great tool. It has saved me a lot of work!  I use sendkeys when creating screen casts and would love for the delay between each keystroke to be in a random range. This would make it look a lot more like a human typing.

I'm going to see if I can write something hacky that inserts some random <p:> directives, but it'd be great for the tool to support it natively.

**Comment by socsieng** [comment]
Might be able to introduce another input argument like `--random-delay`. Where this value is the number of seconds randomised between `0` and random amount to add to the default delay value.

What do you think?

**Comment by cltrudeau** [comment]
Yep, that is what I was thinking. That's how I've seen it done in other similar libraries. Would make an attempt at it myself, but don't do Swift yet... that never ending "yet".
